src/main/java/no/uib/inf112/core/player/PlayerHandler.java

```diff
@@ -116,6 +121,10 @@ public int getPlayerCount() {
         return playerCount;
     }
 
+    public NonPlayer testPlayer() {
+        return (NonPlayer) players.get(0);
+    }
```

Comment:
Do we really need this?

Reply:
Currently yes

Reply:
Do you have any suggestions on what else to do?

Reply:
The only reason to use the NonPlayer is to be able to use the getNextCard(id)? then I think you could just add that to the Player class, since the way things are now every Player is a NonPlayer,

Reply:
I don't think that is a good idea. Because the UserPlayer should not be able to choose cards that way.